packages/ui-extensions/docs/surfaces/customer-account/reference/examples/apis/analytics-publish.example.tsx

```diff
@@ -1,9 +1,10 @@
+import '@shopify/ui-extensions/preact';
 import {render} from 'preact';
 import {useEffect} from 'preact/hooks';
 
-export default function extension() {
+export default async () => {
```

Comment:
Updated all examples to use async anon function since that's what we want to update our CLI template to ([issue](https://github.com/shop/issues-checkout/issues/5915))